setup.cfg

```diff
@@ -20,7 +20,7 @@ ignore = E241,E305,W504
 
 [tool:pytest]
 addopts =
-    --showlocals --durations=20 --doctest-modules -ra --cov-report= --cov=mne
+    --showlocals --durations=20 --doctest-modules -ra --cov-report=
```

Comment:
When running `pytest` locally it would get stuck for several seconds at the end of each run. I realized this was it compiling code coverage estimates. So let's not do it by default (remove it from `setup.cfg`) and instead have the CIs do it explicitly.